downloadGFS.sh

```diff
@@ -183,7 +183,7 @@ downloadGFS()
                     localFileType="$(file $localFile 2>> $SYSLOG)"
```

Comment:
I am going to approve and merge this, but maybe we need a separate utility that does some deep verification if this step is actually needed. E.g., if we're just checking that it's "data" then that doesn't seem to be much different than making sure the file exists or that it's got a size > 0 bytes. Just a thought!